src/pages/en/guides/markdown-content.md

```diff
@@ -364,51 +366,54 @@ The instructions below are for configuring standard Markdown. To configure MDX p
 
 ### Markdown Plugins
 
-Astro supports third-party [remark](https://github.com/remarkjs/remark) and [rehype](https://github.com/rehypejs/rehype) plugins for Markdown. You can provide your plugins in `astro.config.mjs`.
+Astro supports third-party [remark](https://github.com/remarkjs/remark) and [rehype](https://github.com/rehypejs/rehype) plugins for Markdown. These plugins allow you to extend your Markdown with new capabilities, like [auto-generating a table of contents](https://github.com/remarkjs/remark-toc), [applying accessible emoji labels](https://github.com/florianeckerstorfer/remark-a11y-emoji), and more. We encourage you to browse [awesome-remark](https://github.com/remarkjs/awesome-remark) and [awesome-rehype](https://github.com/rehypejs/awesome-rehype) for some full curated lists!
 
-:::note
-Enabling custom `remarkPlugins` or `rehypePlugins` will remove these built-in plugins and you need to explicitly add these plugins if desired.
+You can apply these plugins in your Astro config:
 
-By default, Astro comes with [GitHub-flavored Markdown](https://github.com/remarkjs/remark-gfm) and [remark-smartypants](https://github.com/silvenon/remark-smartypants) pre-enabled.
-:::
+<Tabs client:visible>
+<Fragment slot="tab.1.withDefaults">With defaults</Fragment>
+<Fragment slot="tab.2.withoutDefaults">Without defaults</Fragment>
+<Fragment slot="panel.1.withDefaults">
+
+We apply the [GitHub-flavored Markdown](https://github.com/remarkjs/remark-gfm) and [Smartypants](https://github.com/silvenon/remark-smartypants) remark plugins by default. This brings some niceties like auto-generating clickable links from text (ex. `https://example.com`) and formatting quotes for readability. To apply your own plugins while preserving these defaults, use a nested `extends` object like so:
```

Comment:
Just checking, is `https://example.com` supposed to be clickable when rendered to the Docs page as an example of what actually happens? Because it's not, and that seems... weird? since we're talking about clickable links?

Maybe an example of a link showing up as clickable, and using https://astro.build might make more sense? Or, I'd honestly just lose the example, I think.

Also, I might make it explicit that the example is adding the rehymeMinifyHTML plugin, otherwise it's at first glance a little unintuitive that it is highlighted, vs just the "extends." Maybe something like:

``` 
To apply your own plugins while preserving these defaults, use a nested `extends` object as shown below when adding `rehypeMinifyHtml`:
```

You also have the option of highlighting JUST the extends object, if you like. (I'll let you decide.) Instead of highlighting that whole line, using just quotation marks around the text you'd like highlighted. It might show the object off nicely!